ivy/functional/ivy/general.py

```diff
@@ -2333,7 +2333,30 @@ def num_arrays_in_memory():
 
 
 def print_all_arrays_in_memory():
-    """Prints all arrays which are currently alive."""
+    """
+    Gets all the native Ivy arrays which are currently alive(in the garbage collector)
+    from get_all_arrays_in_memory() function and prints them to the console.
+    
+    Parameters
+    ----------
+    None
+
+    Output
+    ------
+    Type and shape of all the Ivy native arrays which are currently alive.
+
+    Returns
+    -------
+    None
+
+    Example
```

Comment:
I'm going to have to apologise, as this function returns framework specific class objects, we cannot implement and example in this case. If you could please remove that would be great

Reply:
No problem at all! Gimme a minute.

Reply:
Is it okay now?